### PR TITLE
Update dev to Julia v1.6

### PR DESCRIPTION
**Pull request by claireh93**
Follow-up to #10, with a few tweaks for DRU.

### CODE REVIEW

File: .github/workflows/testing.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.4', '1.5']
+        julia-version: ['1.5', '1.6']
```
Comment (richardreeve) [inline .github/workflows/testing.yaml:13]:
```suggestion
        julia-version:
          - '1.4'
          - '1'
```

Comment (richardreeve) [reply .github/workflows/testing.yaml:13]: Don't we want to test against the current LTS, which I *think* is still 1.4, and the latest version, which I think again can be represented as just 1. Unless it actually doesn't work with 1.4? If so, maybe just enforce 1.6 - it'll be the next LTS I think.

File: Project.toml [hunk 2]
```diff
@@ -55,7 +55,7 @@ ArchGDAL = "~0.4.0"
 HDF5 = "^0.15.0"
 JLSO = "2.3"
 Phylo = "^0.4.2"
-julia = "1.1"
+julia = "1.5"
```
Comment (richardreeve) [inline Project.toml:58]: Is there some reason for requiring 1.5? I would go for 1.4 or 1.6 (as above).

**Review by richardreeve — COMMENTED** [review]
Minor issues

File: .github/workflows/testing.yaml (hunk 1 above)
Comment (claireh93) [reply .github/workflows/testing.yaml:13]: It fails on 1.4 because DataRegistryUtils requires 1.5 or higher.

**Comment by claireh93** [comment]
Superseded by #17

**Pull request closed without merging**